src/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="../dist/output.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+      integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="icon" href="../dist/image/google.png" type="google icon" />
+    <title>Google</title>
+  </head>
+  <body class="text-gray-800 text-sm font-Arial">
+    <header class="md:flex md:justify-end w-full">
+      <div class="hidden md:flex space-x-4 p-2 items-center justify-start">
+        <a href="#" class="hover:underline">Gmail</a>
+        <a href="#" class="hover:underline">Images</a>
+        <a href="#" class="hover:bg-gray-200 rounded-full p-4">
+          <img src="../dist/image/dots-menu.png" alt="" />
+        </a>
+        <a href="#" class="hover:bg-gray-200 rounded-full p-2">
+          <img
+            src="../dist/image/account_circle.svg"
+            alt="user"
+            class="w-8 h-8"
+          />
+        </a>
+      </div>
+      <div class="flex justify-between space-x-4 p-2 items-center md:hidden">
+        <div class="text-xs space-x-3 flex items-center">
+          <a href="#">
+            <svg
+              style="fill: #70757a; width: 24px; height: 24px"
+              viewBox="0 0 24 24"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path>
+            </svg>
+          </a>
+          <a href="#" class="text-[#4285f4] border-b-[#4285f4] border-b-2 p-2"
```

Comment:
Instead of using arbitrary values such as text-[#4285f4], add this specific style in the tailwind config file.